backend/inspirehep/records/tasks.py

```diff
@@ -80,11 +78,14 @@ def update_references_pointing_to_merged_record(
     refs_to_schema, merged_record_uri, new_record_uri
 ):
     for index, path in refs_to_schema:
-        config = get_config_for_given_path(index, path)
-        matched_records = match({"$ref": merged_record_uri}, config)
+        query = get_query_for_given_path(index, path, merged_record_uri)
+        es_index_name = f"records-{index}"
```

Comment:
does this take into account the index prefix?

Reply:
yes, otherwise tests would fail cause they're prefixed (I also thought that it wouldn't take prefixes into account, but when I added function `prefix_index` it was prefixed twice)